### DISCUSSION TITLE
課題04についての質問

### DISCUSSION THREAD

**Discussion by TakedaAyano** [nsc-kiwamu/traning-takeda#9] [Q&A]
質問が２点ございます。

①「for文」と「for～of文」の違いがよくわかりませんでした。書き方が違うだけで、内容としては同じなのでしょうか？

②チャレンジ問題については、ネットで調べても書き方がわかりませんでした。
自分で考え書いてみたのですが合っているのでしょうか？
また、配列の中身とその個数を出力させるには、どのようにループさせるのでしょうか？

**Comment by nsc-kiwamu** [comment]
以下で回答済
https://github.com/nsc-kiwamu/traning-takeda/pull/8#issuecomment-1175152453

https://github.com/nsc-kiwamu/traning-takeda/pull/8#issuecomment-1175155432

**Reply by TakedaAyano** [reply]
ご回答ありがとうございます。

string型だけでなく、number型も追加してそこに個数を書いていくということですね、実施してみます。
